src/modules/new-request-form/renderNewRequestForm.tsx

```diff
@@ -3,12 +3,21 @@ import type { Settings } from "../garden-theme/createTheme";
 import { createTheme, ThemeProviders } from "../garden-theme";
 import type { NewRequestFormProps } from "./NewRequestForm";
 import { NewRequestForm } from "./NewRequestForm";
+import { addZendeskTranslations, initI18next } from "../i18n";
 
-export function renderNewRequestForm(
+export async function renderNewRequestForm(
   settings: Settings,
   props: NewRequestFormProps,
   container: HTMLElement
 ) {
+  const { baseLocale } = props;
+
+  initI18next(baseLocale);
+  await addZendeskTranslations(
+    baseLocale,
+    () => import(`./translations/locales/${baseLocale}.json`)
```

Comment:
Why not pass in the json already imported instead of the dynamic import?

Reply:
It is because the `addZendeskTranslations` function contains a try-catch around the dynamic import, to avoid throwing if the requested file is not present.

If we import the file directly here, we need to wrap it in a try-catch. At that point, the `addZendeskTranslations` is more or less useless and we can just move the logic here.

Reply:
Got it! One can argue that it is not a traditional `fetch` call and the file should actually exist and be loaded properly for the app to work as expected.
Where I am getting at is that this might actually be one of the errors that we do want.

Totally fine to keep this behaviour though 👍🏼